src/test/java/uk/ac/ebi/eva/pipeline/runner/EvaPipelineJobLauncherCommandLineRunnerTest.java

```diff
@@ -168,6 +171,63 @@ public void genotypedVcfJobTest() throws JobExecutionException, IOException, URI
     }
 
     @Test
+    public void doNotReuseParametersFromPreviousCompletedJobs() throws JobExecutionException, IOException, URISyntaxException,
+            ClassNotFoundException, StorageManagerException, InstantiationException, IllegalAccessException {
+        String databaseName = mongoRule.getRandomTemporaryDatabaseName();
+        File inputFile = GenotypedVcfJobTestUtils.getInputFile();
+
+        evaPipelineJobLauncherCommandLineRunner.setJobNames(GENOTYPED_VCF_JOB);
+        EvaCommandLineBuilder evaCommandLineBuilderWithoutChunksize = new EvaCommandLineBuilder()
+                .inputVcf(inputFile.getAbsolutePath())
+                .inputVcfAggregation("NONE")
+                .inputStudyName("small vcf")
+                .inputStudyId(GenotypedVcfJobTestUtils.INPUT_STUDY_ID)
+                .inputStudyType("COLLECTION")
+                .databaseName(databaseName)
+                .configDbReadPreference("secondary")
+                .dbCollectionsVariantsName("variants")
+                .dbCollectionsFilesName("files")
+                .dbCollectionsFeaturesName("features")
+                .annotationSkip(true)
+                .statisticsSkip(true);
+
+        String[] commandLineWithoutChunksize = evaCommandLineBuilderWithoutChunksize
+                .inputVcfId("second_" + GenotypedVcfJobTestUtils.INPUT_VCF_ID)
+                .build();
+
+        String[] completeCommandLine = evaCommandLineBuilderWithoutChunksize
+                .inputVcfId("first_" + GenotypedVcfJobTestUtils.INPUT_VCF_ID)
+                .chunksize("100")
+                .build();
```

Comment:
If this is the first one, could it be declared first? Otherwise it looks like an oversight that could involve weird behavior in the test.